base/src/test/java/com/mindera/skeletoid/logs/LOGUnitTest.java

```diff
@@ -426,6 +531,64 @@ public void testVerboseLog() {
         verify(appenderC, times(1)).log(LOG.PRIORITY.VERBOSE, null, log);
     }
 
+    @Test
+    public void testRemoveAllAppenders() {
+        Context context = mock(Context.class);
+        List<ILogAppender> appenders = new ArrayList<>();
+        ILogAppender appenderA = mockAppender("A");
+        ILogAppender appenderB = mockAppender("B");
+        ILogAppender appenderC = mockAppender("C");
+        appenders.add(appenderA);
+        appenders.add(appenderB);
+        appenders.add(appenderC);
+
+        LOG.init(context, mPackageName);
+        final Set<String> ids = LOG.addAppenders(context, appenders);
+
+        LOG.removeAppenders(context, ids);
+
+        verify(appenderA).disableAppender();
+        verify(appenderB).disableAppender();
+        verify(appenderC).disableAppender();
+    }
+
+    @Test
+    public void testRemoveSomeAppenders() {
+        Context context = mock(Context.class);
+        List<ILogAppender> appenders = new ArrayList<>();
+        ILogAppender appenderA = mockAppender("A");
+        ILogAppender appenderB = mockAppender("B");
+        ILogAppender appenderC = mockAppender("C");
+        appenders.add(appenderA);
+        appenders.add(appenderB);
+        appenders.add(appenderC);
+
+        LOG.init(context, mPackageName);
+        final Set<String> ids = LOG.addAppenders(context, appenders);
+        final List<String> idsList = new ArrayList<>(ids);
+
+        for (String id : ids) {
+            if (id.equals("A")) {
+                idsList.remove(id);
+            }
+        }
+        final Set<String> idsToRemove = new HashSet<>(idsList);
+
+        LOG.removeAppenders(context, idsToRemove);
+
+        verify(appenderA, times(0)).disableAppender();
+        verify(appenderB).disableAppender();
+        verify(appenderC).disableAppender();
+    }
+
+    @Test
+    public void testRemoveNullAppenders() {
+        Context context = mock(Context.class);
+        LOG.init(context, mPackageName);
+
+        LOG.removeAppenders(context, null);
```

Comment:
I think im missing something, shouldn't we be verifying or asserting something here ?